src/DynamoCore/Graph/Workspaces/PackageInfo.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Dynamo.Graph.Workspaces
+{
+    /// <summary>
+    /// Class containing info about a Dynamo package. 
+    /// Used for serialization.
+    /// </summary>
+    internal class PackageInfo
+    {
+        /// <summary>
+        /// Name of the package
+        /// </summary>
+        internal string Name { get; set; }
+
+        /// <summary>
+        /// Version of the package
+        /// </summary>
+        internal string Version { get; set; }
+
+        /// <summary>
+        /// The full name of this package, including its name and version
+        /// Format: name_version
+        /// </summary>
+        internal string FullName { get { return Name + "_" + Version; } }
+
+        /// <summary>
+        /// Guids of nodes in the workspace that are dependent on this package
+        /// </summary>
+        internal List<Guid> Dependents
```

Comment:
hmm, DependentNodes?

Reply:
can you use an auto implemented property for this or access modifiers make it tough?

Reply:
Scott likes Nodes